modules/github_integration/lib/open_project/github_integration/engine.rb

```diff
@@ -85,9 +85,13 @@ class Engine < ::Rails::Engine
       mount ::API::V3::GithubPullRequests::GithubPullRequestsAPI
     end
 
-    config.to_prepare do
-      # Register the cron job to clean up old github pull requests
-      ::Cron::CronJob.register! ::Cron::ClearOldPullRequestsJob
+    add_cron_jobs do
+      {
+        ClearOldPullRequestsJob: {
+          cron: '25 1 * * *', # runs at 1:25 nightly
+          class: 'ClearOldPullRequestsJob'
```

Comment:
Same question here if we could use the class literal to make sure we notice if the class is renamed. So `class: ClearOldPullRequestsJob.name`.

Reply:
Yes, that makes perfect sense. Will do.